packages/table/src/interactions/selectable.tsx

```diff
@@ -93,6 +101,27 @@ export class DragSelectable extends React.Component<IDragSelectableProps, {}> {
             return false;
         }
 
+        const regionCardinality = Regions.getRegionCardinality(region);
+        let focusCellCoordinates: IFocusedCellCoordinates = null;
+
+        if (regionCardinality === RegionCardinality.FULL_TABLE) {
+            focusCellCoordinates = { col: 0, row: 0 };
+        }
+
+        if (regionCardinality === RegionCardinality.FULL_COLUMNS) {
+            focusCellCoordinates = { col: region.cols[0], row: 0 };
```

Comment:
You can't focus the header cells (just like in excel, etc). It'll return the top-left cell, in the body, that corresponds with the thing you just clicked on. Same behavior as excel, etc.